tasks/internal/push-artifacts-to-cdn-task/push-artifacts-to-cdn-task.yaml

```diff
@@ -710,7 +715,7 @@ spec:
             echo "Executing SSH command with sign method: $sign_method"
             # shellcheck disable=SC2029,SC2086
             ssh $SSH_OPTS "$checksum_user@$checksum_host" \
-            "rpm-sign --nat $sign_method --key redhatrelease2 --onbehalfof=$AUTHOR \
```

Comment:
Wouldn't it make sense to have `redhatrelease2` as the default and not require it to be always set? Just curious.

Reply:
Also, wouldn't it be easier to have the signing key id in the data directly, rather than having a configmap name and read that? Do we anticipate having more things in the configmap in the future?

Reply:
The configmaps already exist and are used elsewhere https://github.com/hacbs-release/app-interface-deployments/tree/main/internal-services/config/signing/production

Reply:
Ah, ok, makes sense, thanks.